ckanext/datastore/plugin.py

```diff
@@ -390,8 +390,24 @@ def datastore_search(self, context, data_dict, fields_types, query_dict):
         sort = self._sort(data_dict, fields_types)
         where = self._where(data_dict, fields_types)
 
-        select_cols = [
-            datastore_helpers.identifier(field_id) for field_id in field_ids]
+        select_cols = []
+        records_format = data_dict.get(u'records_format')
+        json_values = records_format in (u'objects', u'lists')
+        for field_id in field_ids:
+            fmt = u'to_json({0})' if records_format == u'lists' else u'{0}'
```

Comment:
@amercader Looks like this change requires postgres 9.3.  Should I rewrite it with the functions available in 9.2 or should we bump the postgres version requiremnet?